datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/UpdateNameResolver.java

```diff
@@ -98,4 +101,28 @@ private Boolean updateGlossaryNodeName(
     }
     throw new AuthorizationException("Unauthorized to perform this action. Please contact your DataHub administrator.");
   }
+
+  private Boolean updateDomainName(
+      Urn targetUrn,
+      UpdateNameInput input,
+      QueryContext context
+  ) {
+    if (AuthorizationUtils.canManageGlossaries(context)) {
```

Comment:
Is this right? We should be checked whether you can manage domains (i guess)

Reply:
no you're correct that's my fault